arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -176,7 +159,7 @@
             "description": "The requested automatic tuning resource state.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/AutomaticTuningUpdate"
+              "$ref": "#/definitions/AutomaticTuning"
```

Comment:
Server level automatic tuning is still under development, so I will add x-ms-examples when I have a working prototype. Database level automatic tuning is fully functional, and those examples are generated from real request/response